pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -636,10 +636,7 @@ func setPasswords(cr *api.KieApp, isTrialEnv bool) {
 		&cr.Spec.CommonConfig.AdminPassword,
 		&cr.Spec.CommonConfig.DBPassword,
 		&cr.Spec.CommonConfig.AMQPassword,
-		&cr.Spec.CommonConfig.AMQClusterPassword,
-		&cr.Spec.CommonConfig.ControllerPassword,
-		&cr.Spec.CommonConfig.MavenPassword,
-		&cr.Spec.CommonConfig.ServerPassword}
```

Comment:
do we need to add some logic here that still adds these passwords if the version is less than 7.7.0? I assume we'll still need these passwords for older product versions?

maybe something like
```shell
	if getMinorImageVersion(cr.Spec.Version) < "77" {
               passwords = append(passwords, &cr.Spec.CommonConfig.ControllerPassword) ...
....
        }
```
???

Reply:
The templates for 7.5.1 and 7.6.0 have the variables set so I think we can just delete them once the oldest templates are 7.7.0

Reply:
i think that makes sense ya... so, for this change then, would you add them back to this slice or?

Reply:
Of course, I totally forgot.